hm3.js

```diff
@@ -0,0 +1,53 @@
+// 1
+var count = window.prompt('Введите число: ', 0);
+
+
+
+nextPrime:
+  for (var i = 2; i <= count; i++) {
+
+    for (var j = 2; j < i; j++) {
+      if (i % j == 0) continue nextPrime;
+    }
+
+    console.log( i ); // простое
+  }
+
+
+// 2
+
+console.log("Ответ на 5 задание");
+    
+    var arr = [];
+    var end = 0;
+    while (end < 20){
+        end++;
+        arr.push('x');
+        console.log(arr);
+    }
+
+//4 - не работает
+// var n = 10;
+// var i;
+// do { 
+// 	console.log(i++)
+// } for (i = 0; i < n; i++) {};
+
+//2,3
+var basket_price = 0;
+
+Basket = [{name:'Футболка', price:100}, {name:'Шорты', price: 50}]
+
+function countBasketPrice(x){
+	for (i=0; i<Basket.lenght; i++){

```

Comment:
тут была опечатка, потому и не считалось
```suggestion
	for (i=0; i<Basket.length; i++){
```